apps/frontend/src/app/pages/home/home.component.ts

```diff
@@ -14,7 +15,13 @@ import { LocalUserService } from '../../services/data/local-user.service';
 export class HomeComponent {
   user: User;
 
-  constructor(private readonly localUserService: LocalUserService) {
+  constructor(
+    private readonly localUserService: LocalUserService,
+    private readonly router: Router
+  ) {
+    if (!this.localUserService.isLoggedIn()) {
```

Comment:
You're calling a regular just once from the constructor here. The only reason it works is because the login checks are syncronous and so happen before this constructor is called.

Better to subscribe to the loginchanged behavioursubject, and do the redirect it emits with `false`. That way if auth ever becames asyncronous, or you do a logout whilst on this page, it'll redirect.